### ISSUE TITLE
Question about the Qformer training

### ISSUE THREAD

**Issue by TobiasLee** [salesforce/LAVIS#250]
Hi, big thanks for your great work and open-sourced code & weights.
I am fine-tuning/continuing training the checkpoints and hope you can kindly share some knowledge about the design & training details:

1. what's the average loss of the final checkpoint for OPT decoder, i.e., the captioning loss. I'd like to use this to check whether my model is converging. 

2. did u have a plan to release some checkpoints of the first stage pre-training, as the Qformer trained after the first stage is model-agnostic?

**Comment by zzhanghub** [comment]
The ckpt of first stage pre training would be very helpful. I also look forward to its release.

**Comment by LiJunnan1992** [comment]
We have released stage-1 checkpoints with both [ViT-g](https://github.com/salesforce/LAVIS/blob/main/lavis/configs/models/blip2/blip2_pretrain.yaml) and [ViT-L](https://github.com/salesforce/LAVIS/blob/main/lavis/configs/models/blip2/blip2_pretrain_vitL.yaml).

You could run BLIP-2 stage1 pre-training from scratch now with `bash run_scripts/blip2/train/pretrain_stage1.sh`

Thank you.
